sqlconnect/internal/bigquery/db.go

```diff
@@ -28,7 +28,13 @@ func NewDB(configJSON json.RawMessage) (*DB, error) {
 		return nil, err
 	}
 
-	db := sql.OpenDB(driver.NewConnector(config.ProjectID, option.WithCredentialsJSON([]byte(config.CredentialsJSON))))
+	db := sql.OpenDB(driver.NewConnector(
+		config.ProjectID,
+		driver.Config{
+			JobRateLimitExceededRetryEnabled: true,
```

Comment:
**Note:** enabling it and not giving an option to disable it, since this really a BigQuery client bug. The reason why a configuration option for it exists in the driver itself is for verifying the issue and its fix in tests.